.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies)_.
+
+### Affected node.js versions
+
```

Comment:
Perhaps add a statement asking the reporter to fill in the specific minor and patch level also